test/test_bdecode.cpp

```diff
@@ -949,7 +953,8 @@ TORRENT_TEST(dict_find_funs2)
 
 	// try finding the last item in a dict (to skip all the other ones)
 	TEST_EQUAL(e.dict_find("d").type(), bdecode_node::dict_t);
-	TEST_EQUAL(e.dict_find(std::string("d")).type(), bdecode_node::dict_t);
+	std::string d_str("d");
+	TEST_EQUAL(e.dict_find(d_str).type(), bdecode_node::dict_t);
```

Comment:
here it also made for testing of passing std::string as argument (or else it must be completly removed cause line above do the same test but with "d" passed as const literal)